src/xdocs/checks/design/finalclass.xml

```diff
@@ -15,7 +15,7 @@
         </p>
         <ol>
           <li>
-              Private classes with implicit constructor.
+              Private classes with no declared constructors.
```

Comment:
Why was this changed? We had no issues merging this just 2 weeks ago.

Reply:
Uh, I forgot to ask about this one. The validation wasn't there 2 weeks ago so this one slipped.
@Kevin222004 Could you please tell which one is the true one(the one that should stay)?
> Private classes with implicit constructor.

or

> Private classes with no declared constructors.

Reply:
@stoyanK7 this is the true one
`Private classes with no declared constructors`  this statment should stay.

usually I always run mvn clean verify but 

>  The validation wasn't there 2 weeks ago so this one slipped.

due to this may be i also missed

Reply:
Thanks. We shouldn't disable this validation unless there is some replacement.